frontend/public/components/_quota.scss

```diff
@@ -5,3 +5,65 @@
 .co-resource-quota-empty {
   font-size: 15px;
 }
+
+.co-resource-quota-chart-row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+  .co-resource-quota-chart {
```

Comment:
Here and throughout:  BEM suggests to avoid nesting.  It's ok to use it if you're dealing with external dependency rules you need to scope, but this case and others below is not that.

Reply:
Hi @rhamilto, what is the best practice then for overriding classes of a shared component then?  Here, `<QuotaGaugeCharts ...` has a parent class of `quota-dashboard-row` which has all those overrides/nesting of `group__graphs` and `graph-wrapper` to make the charts more condensed when in the dashboard, and overrides of `co-resource-quota-scope__label` & `....__description` so that they appear side-by-side when in the dashboard.  In another area of the gui `<QuotaGaugeCharts ...`  doesn't have the parent `quota-dashboard-row` so those charts have more space, line returns, etc..  Is it recommended I pass in a `className` prop, or do something with default values? -thanks

Reply:
BEM suggest using classes as needed.  If your element needs to be modified from a default, then a modifier class is added.  This breaks the dependency on parent nodes.  http://getbem.com/naming/ has good examples.